task/hiffy/src/common.rs

```diff
@@ -514,6 +518,236 @@ pub(crate) fn spi_write(
     Ok(0)
 }
 
+#[cfg(feature = "sprot")]
+task_slot!(SPROT, sprot);
+
+#[cfg(feature = "sprot")]
+pub(crate) fn sprot_send_recv(
```

Comment:
There are warnings here about unused functions, can you double check config settings?